NEWS.md

```diff
@@ -2,6 +2,9 @@
 
 * `new_quant_param()` no longer requires `range` and `inclusive` if `values` is
   supplied (#87).
+  
+* `mtry_prop()` is a new parameter for use with engines that interpret `mtry()`
+  as a proportion rather than a count in tree-based models (@simonpcouch, #233).
```

Comment:
I think in practice we only add the PR author here if they are not part of the team already but you're welcome to keep it in!